src/style.css

```diff
@@ -1,3 +1,69 @@
+* {
+  margin: 0;
+  box-sizing: border-box;
+}
+
 body {
+  --text-hover-color: #00acdc;
+  font-family: "Roboto", sans-serif;
+}
+
+.logo__img {
+  padding: 10px 0 10px 50px;
+}
+
+.nav {
+  position: absolute;
+  right: 50px;
```

Comment:
Position absolute looks redundant here, it's better to add display: flex and justify-content: space-between rules to `header` class